custom/icds_reports/utils/aggregation.py

```diff
@@ -703,3 +703,206 @@ def recalculate_aggregate_table(model_class):
 
         for month in range(1, date.today().month + 1):
             model_class.aggregate(state_id, date(2018, month, 1))
+
+
+class ChildHealthMonthlyAggregationHelper(BaseICDSAggregationHelper):
+    aggregate_parent_table = AGG_GROWTH_MONITORING_TABLE
+    aggregate_child_table_prefix = 'icds_db_gm_form_'
```

Comment:
are these supposed to all still say growth monitoring?

Reply:
Removed in the latest commit. Those aren't actually used because this is still using hte manual partitioning that was set up instead of the new style that partitions based on state & month